Owncloud iOs Client/Network/Downloads/Download.m

```diff
@@ -434,6 +436,9 @@ - (void)updateDataDownload{
     
     _fileDto.isNecessaryUpdate = isNecessaryUpdate;
     
+    ManageThumbnails *manageThumbnails = [ManageThumbnails sharedManager];
```

Comment:
We should use directly the singleton not getting the singleton in a local instance. Just to use the same style on all the app (like the other singletons) :wink: